src/engine/Collision/EdgeArea.ts

```diff
@@ -79,7 +79,7 @@ export class EdgeArea implements ICollisionArea {
    /**
     * Tests if a point is contained in this collision area
     */
-   public contains(point: Vector): boolean {
```

Comment:
I think we should keep this point, that way we keep a consistent interface with all other collision areas. Otherwise maybe just axe the method entirely since EdgeArea cannot contain any points.